app/core/config.py

```
@@ -16,52 +42,122 @@ class BaseConfig:
-elif ENV == "dev":
-    configs = DevelopConfig()
+if ENV == "prod":
+    pass
 elif ENV == "stage":
-    configs = StageConfig()
-elif ENV == "prod":
-    configs = ProductionConfig()
-else:
-    logger.info("ENV is not set. It will be set to 'BaseConfig'.")
\ No newline at end of file
+    pass
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)